### DISCUSSION TITLE
Imgbot cannot be re-installed!

### DISCUSSION THREAD

**Discussion by t1noo7** [community/community#140391] [Pull Requests]
Recently updated: I just got the 1st issued, but does anyone like me: when installed it, tried waiting for so long but no PR arrive! :(

Hi everyone,

I have 2 problems with this topic :P

First, I've installed the Imgbot app recently here, as it's instructed, the bot will automatically send an optimization pull request, but nearly one week passed, and nothing to show (no request, nothing). I found it strange and tried uninstalling and reinstalling. And now getting this: 

<img src = "https://github.com/user-attachments/assets/bc5e2b20-1e5c-4dfa-aa93-599a19a53f80" />

I tried to scan the repos and go to settings to check and remove all traces of ImgBot, also removed in general settings -> application -> Installed GitHub Apps/Authorized GitHub Apps/Authorized OAuth Apps, removed and revoked cleanly but still got the above message. Go to ImgBot's homepage and get the same image above, can't reinstall it, I played something stupid because I don't know the right, guys. What am I supposed to do :(

**Comment by t1noo7** [comment]
Oh, I got it myself :P

**Reply by snehilshah** [reply]
Hey, what solved your problem?
